lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -2628,6 +3109,10 @@ ActorIsolation ActorIsolationRequest::evaluate(
   if (auto nominal = value->getDeclContext()->getSelfNominalTypeDecl()) {
     if (nominal->isActor() &&
         (value->isInstanceMember() || isa<ConstructorDecl>(value))) {
+      defaultIsolation = nominal->isDistributedActor() ?
+                         ActorIsolation::forDistributedActorInstance(nominal) :
+                         ActorIsolation::forActorInstance(nominal);
+    } else if (isa<ConstructorDecl>(value)) {
       defaultIsolation = ActorIsolation::forActorInstance(nominal);
```

Comment:
I think this is causing the error. It seems we are now treating every constructor as actor isolated, which would explain the `hop_to_executor` onto `Float80`.

```suggestion
```

Reply:
Thank you for digging in! let me check that